src/test/kotlin/org/phoenixframework/PhxSocketTest.kt

```diff
@@ -36,4 +41,41 @@ class PhxSocketTest {
         assertThat(PhxSocket("wss://localhost:4000/socket/websocket", spacesParams).endpoint.toString())
                 .isEqualTo("https://localhost:4000/socket/websocket?user_id=1&token=abc%20123")
     }
+
+    @Test
+    fun isConnected_isTrue_WhenSocketConnected() {
+        val mockSocket = Mockito.mock(WebSocket::class.java)
+        val socket = PhxSocket("http://localhost:4000/socket/websocket")
+
+        socket.onOpen(mockSocket, null)
+
+        assertTrue(socket.isConnected)
```

Comment:
This project uses Google's `Truth` lib for assertions

```
assertThat(socket.isConnect).isTrue()
```

Reply:
Missed that. Updated now.